upup/pkg/fi/cloudup/azuretasks/loadbalancer.go

```diff
@@ -151,6 +179,7 @@ func (*LoadBalancer) RenderAzure(t *azure.AzureAPITarget, a, e, changes *LoadBal
 			ID: to.StringPtr(fmt.Sprintf("/%s/virtualNetworks/%s/subnets/%s", idPrefix, *e.Subnet.VirtualNetwork.Name, *e.Subnet.Name)),
 		}
 	}
+	// TODO: Move hardcoded values to the model
```

Comment:
Almost everything about LB is hardcoded. We should move this to model soon. For now it is good enough.